project/project-grader.json

```diff
@@ -3,7 +3,7 @@
   "code": "ready!set!responsiveimages!",
   "tests": [
     {
-      "description": "&lt;img&gt;s have max-widths of 100%",
+      "description": "<img> have max-widths of 100%",
```

Comment:
@notetiene are you parsing descriptions differently now? I went the &lt; and &gt; approach because angle brackets were getting rendered as HTML